report/26_Process-Development-Process-&-Tools.md

```diff
@@ -1,2 +1,52 @@
-Applied development process and tools supporting it
-For example, how did you use issues, Kanban boards, etc. to organize open tasks
+### Applied development process and tools supporting it
+
+Throughout this course, two tools, Github and Discord, has been used to manage the project and communication.
+Two tools have been used, to differentiate between synchronous and asynchronous communication.
```

Comment:
You also write that two tools have been used right above. These two sentences right after each other sound super weird